apmpackage/apm/data_stream/internal_metrics_transaction/fields/fields.yml

```diff
@@ -0,0 +1,85 @@
+- name: faas.coldstart
+  type: boolean
+  description: |
+    Boolean indicating whether the function invocation was a coldstart or not.
+- name: faas.id
+  type: keyword
+  description: |
+    A unique identifier of the invoked serverless function.
+- name: faas.execution
+  type: keyword
+  description: |
+    Request ID of the function invocation.
+- name: faas.trigger.request_id
+  type: keyword
+  description: |
+    The ID of the origin trigger request.
+- name: faas.trigger.type
+  type: keyword
+  description: |
+    The trigger type.
```

Comment:
I don't think these are relevant either

Reply:
`faas.trigger.type` is used, but the other two aren't, I'll remove them.